libobs/CMakeLists.txt

```diff
@@ -369,7 +357,6 @@ set(libobs_util_SOURCES
 set(libobs_util_HEADERS
 	util/curl/curl-helper.h
 	util/sse-intrin.h
-	util/sse2neon.h
```

Comment:
it removes references to `sse2neon` but doesnt remove the header.

Reply:
Fixed, thanks!